src/kubernetes-ingress-controller/guides/getting-started.md

```diff
@@ -1,73 +1,42 @@
 ---
 title: Getting started with the Kubernetes Ingress Controller
+content_type: tutorial
 ---
 
-## Installation
+## Overview
 
-Please follow the [deployment](/kubernetes-ingress-controller/{{page.kong_version}}/deployment/overview) documentation to install
-the {{site.kic_product_name}} onto your Kubernetes cluster.
+This guide walks through setting up an HTTP(S) route and plugin using
+{{site.base_gateway}} and {{site.kic_product_name}}.
 
-## Testing connectivity to Kong
+{% include_cached /md/kic/installation.md kong_version=page.kong_version %}
 
-This guide assumes that `PROXY_IP` environment variable is
-set to contain the IP address or URL pointing to Kong.
-If you've not done so, please follow one of the
-[deployment guides](/kubernetes-ingress-controller/{{page.kong_version}}/deployment/overview) to configure this environment variable.
+{% include_cached /md/kic/http-test-service.md kong_version=page.kong_version %}
 
-If everything is setup correctly, making a request to Kong should return back
-a HTTP `404 Not Found` status code.
+{% include_cached /md/kic/class.md kong_version=page.kong_version %}
 
-```sh
-curl -i $PROXY_IP
-```
-
-In this document, the expected output follows each command:
-```text
-HTTP/1.1 404 Not Found
-Content-Type: application/json; charset=utf-8
-Connection: keep-alive
-Content-Length: 48
-X-Kong-Response-Latency: 0
-Server: kong/2.8.1
-
-{"message":"no Route matched with those values"}
-```
-
-This is expected since Kong doesn't know how to proxy the request yet.
+## Add routing configuration
 
-## Set up an echo-server
-
-Setup an echo-server application to demonstrate how
-to use the {{site.kic_product_name}}:
-
-```sh
-kubectl apply -f https://bit.ly/echo-service
-```
-
-```text
-service/echo created
-deployment.apps/echo created
-```
+Create routing configuration to proxy `/echo` requests to the echo server:
```

Comment:
Left as-is. This is somewhat awkward phrasing to work around not having a proper name that applies to both: you will create either an Ingress or an HTTPRoute, and both Ingresses and HTTPRoutes are routing configuration, but there isn't actually _a_ routing configuration, if that makes sense.

There's maybe some better way to phrase around that ambiguity, though IDK what without some sort of page-level templating to avoid the tabs (e.g. you access `getting-started.md?api=Gateway` and in the template it sets up `if page.api == Gateway` then `assign httpResourceName = HTTPRoute` and so on, and only renders the commands for your chosen API).